bats/katello-bats

```diff
@@ -34,8 +34,8 @@ for arg do
       swapfile)
         swapfile=true
         ;;
-      capsule)
-        katello_capsule=true
+      proxy)
+        katello_proxy=true
```

Comment:
I think this case should just be proxy, in line with the filename. On the other hand I'd also like to kill this file (like https://github.com/Katello/forklift/pull/334 helps to do)